src/dom-to-image-more.js

```diff
@@ -558,6 +563,115 @@
             return node;
         });
     }
+
+    function compressSvg(clone) {
+        const sandboxWindow = ensureSandboxWindow();
+        sandboxWindow.document.body.appendChild(clone);
+    
+        // Generate ascending DOM tree for reverse level order traversal.
+        // CSS inheritance is computed downward (preorder traversal) and is additive-cumulative.
+        // The filter op is subtractive and goes upward (to only splice out inheritable style declarations).
+        const walker = document.createTreeWalker(clone, NodeFilter.SHOW_ELEMENT);
+        const tree = [walker.currentNode];
+        let node;
+        while ((node = walker.nextNode())) tree.push(node);
```

Comment:
Can we compute the depth here (and hopefully short circuit to avoid crawling back up multiple times through the same parent-hierarchy?)

Reply:
wow, simple and elegant 😲🤯 will do!

Reply:
Took a shot at it - [59df18e](https://github.com/1904labs/dom-to-image-more/pull/135/commits/59df18e0c9603f66a8fef8568365078f0da562d7)